docs/API_Reference.md

```diff
@@ -638,3 +643,259 @@ ehsm-core enclave will decrypt user-supplied ciphertextblob with specified CMK t
   ```
   *(return to the [Cryptographic Functionalities APIs](#eHSM-REST-API-Reference).)*
 ---
+
+## ListKey
+Query all CMKs of the current appid.
+
```

Comment:
Query all the CMKs generated by the current account.

Reply:
It will be fix